hbase-asyncfs/src/main/java/org/apache/hadoop/hbase/io/asyncfs/FanOutOneBlockAsyncDFSOutputHelper.java

```diff
@@ -640,8 +639,9 @@ public static boolean shouldRetryCreate(RemoteException e) {
 
   static void completeFile(FanOutOneBlockAsyncDFSOutput output, DFSClient client,
     ClientProtocol namenode, String src, String clientName, ExtendedBlock block,
-    HdfsFileStatus stat) {
-    for (int retry = 0;; retry++) {
+    HdfsFileStatus stat) throws IOException {
+    int maxRetries = client.getConf().getBlockWriteLocateFollowingInitialDelayMs();
```

Comment:
The method name refers to milliseconds, which makes this sound we're using a temporal value for the number of retries.

Is this really the config option we want here ? Is the Hadoop getter misnamed ?

Reply:
Oh, should be a typo, there is a retry number config.

Let me fix.